soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -2535,6 +2535,13 @@ namespace Settings {
     } else {
         GanonsTrialsCount.SetSelectedIndex(cvarSettings[RSK_TRIAL_COUNT]);
     }
+    if (cvarSettings[RSK_RANDOM_MQ_DUNGEONS] == 2) {
+        MQDungeonCount.SetSelectedIndex(13);
+    } else if (cvarSettings[RSK_RANDOM_MQ_DUNGEONS] == 0) {
+        MQDungeonCount.SetSelectedIndex(0);
+    } else {
+        MQDungeonCount.SetSelectedIndex(cvarSettings[RSK_MQ_DUNGEON_COUNT]);
+    }
```

Comment:
there's a [PR out](https://github.com/HarbourMasters/Shipwright/pull/1761) for moving away from magic numbers for RSKs to using enum values, it hasn't been merged yet but i figured it's worth adding a note to so we don't forget to add these in there